Marsiple/App/Tab Bar/TabBarController.swift

```diff
@@ -11,18 +11,24 @@ import UIKit
 class TabBarController: UITabBarController {
     init() {
         super.init(nibName: nil, bundle: nil)
-        let viewControllerList = [PostsViewController(), AlbumsViewController()]
+        setupViews()
+    }
+        
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+private extension TabBarController {
+    func setupViews() {
         tabBar.barTintColor = .martianDark
         tabBar.tintColor = .white
```

Comment:
What do you think about extracting these two tab bar related lines into a separate method `setupTabBar`?

Reply:
That's ok, but aren't other lines are also related to `TabBar`?